app/services/mandataris-api.js

```diff
@@ -104,4 +106,66 @@ export default class MandatarisApiService extends Service {
       include: 'fractietype',
     });
   }
+
+  async downloadAsCsv({
```

Comment:
why not just use an html-5 download link? this is the old way of doing things from before this existed but browser support seems good now

Reply:
Can you add a json body to this?